package/doc/sphinx/source/documentation_pages/loading_files/lammps.rst

```diff
@@ -0,0 +1,22 @@
+.. _load_lammps:
+
+####################################
+Loading Lammps files with MDAnalysis
+####################################
+
+.. _load_data:
+
+Loading Data files
+------------------
+
+attributes provided etc
+
+see :mod:`MDAnalysis.topology.LAMMPSParser`
+
+
+.. _load_lammps_dcd:
+
+Loading DCD files
+-----------------
+
```

Comment:
Need to say something about units and how to set them.

Reply:
Tbh I don't really understand how units work.  If there's existing docs I'll copy them over but I won't be able to write these myself

Reply:
I'll have a look... need to remind myself.